src/main/java/frc/robot/wrist/WristPosition.java

```diff
@@ -6,7 +6,8 @@
 
 public enum WristPosition {
   UP(0.25),
-  DOWN(-0.25);
+  OUTAKING(-0.25),
```

Comment:
```suggestion
  OUTTAKING(-0.25),
```
outtaking isn't a real word, but if it were it would be spelled with two Ts